addon/utils/serialize-and-push.js

```diff
@@ -0,0 +1,10 @@
+import { isArray } from '@ember/array';
+
+export default function(options, response) {
```

Comment:
It appears that this options object is not used. Can it be removed?

Reply:
Yes I've removed to the options param